laddertools/ladder.py

```diff
@@ -253,13 +255,99 @@ def run():
     parser.add_argument('--start')
     parser.add_argument('--end')
     parser.add_argument('--bans-file')
+    parser.add_argument('-l', '--log-level', default="WARNING")
     parser.add_argument('replays', nargs='*')
     args = parser.parse_args()
 
+    logging.basicConfig(level=args.log_level)
+
     lockfile = args.database + '.lock'
     lock = FileLock(lockfile, timeout=1)
     try:
         with lock:
             _main(args)
     except Timeout:
         logging.error('Another instance of this application currently holds the %s lock file.', lockfile)
+
+
+def initialize_periodic_databases():
+    """A CLI tool to create multiple database files in batch
+
+    In the current implementation, this is customized to support 2-months-periods as used by the oraladder.net website.
+    For a given timespan based on "year" and "start-month" parameters, SQLite database files get created. Each DB file
+    contains data for a 2-month period starting with ever 2nd month of a year (i.e. January, March, May, ...) which
+    will be named following the pattern db-{year}-{nr}.sqlite3 where {nr} is an integer counter starting with 1 for the
+    Jan-Feb period going up to 6 for the Nov-Dec period of the given year.
+
+    If invoked with a start month that does not mark the beginning of one of these periods, it will get corrected by
+    subtracting one month so that the resulting DB files will be true to the defined format/content.
+
+    For less customized database file creation, refer to the `ora-ladder` CLI tool utilizing "start" and "end"
+    parameters.
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-s', '--schema', default=op.join(op.dirname(__file__), 'ladder.sql'))
+    parser.add_argument('-r', '--ranking', choices=ranking_systems.keys(), default='trueskill')
+    parser.add_argument('--bans-file')
+    parser.add_argument('-m', '--mod', default='ra')
+    parser.add_argument('-y', '--year', default=datetime.date.today().year)
+    parser.add_argument('--start-month', default="1")
```

Comment:
nit: you're mixing quotation

BTW; you may want to run `black` (and `isort`) and add a CI check at some point; would prevent these kind of issues

Also, you could use `type=int`, that would simplify a bit the code.

You may also want to have a `help=...` stating that value need to be between 1 and 12.